src/fancyprint/printer.py

```diff
@@ -284,63 +178,60 @@ def print(self, text: str, align="center", blank_character=" ", left_delimiter=T
 
                 finalized_text_buffer.append(res)
 
-        for ftext in finalized_text_buffer:
-            if len(ftext) > 5:
-                is_color, color_count = self.__check_color_string_in_dict(ftext, Fore.__dict__, 5)
-                if is_color:
-                    length_of_text = int(len(ftext) - (color_count * 5))
-                else:
-                    length_of_text = len(ftext)
+        for finalized_text in finalized_text_buffer:
+            if len(finalized_text) > 5:
+                is_color, color_count = self.__check_color_string_in_dict(finalized_text, Fore.__dict__, 5)
+                length_of_text = len(finalized_text) if not is_color else len(finalized_text) - color_count
             else:
-                length_of_text = len(ftext)
+                length_of_text = len(finalized_text)
 
             # Calculate the amount of space, taking alignment into consideration
-            if align == "center":
+            if config.align == Align.CENTER:
                 # Amount of space is the half of the screen width - the length of the side separators - the half of the text length
                 amount_of_space = int((terminal_size_h - 2) / 2 - length_of_text / 2)
 
```

Comment:
It would be better if alignment is passed as a parameter, as it would be easier to have different alignments in a block.